int32 ticks_L
int32 ticks_R
float32 duty_cycle_L
float32 duty_cycle_R
float32 desr_rpm_L
float32 desr_rpm_R
float32 curr_rpm_L
float32 curr_rpm_R